lib/galaxy/selenium/navigation.yml

```diff
@@ -583,6 +583,8 @@ upload:
       selector: '.selection-dialog-modal'
     rule_dataset_selector_row:
       selector: '.selection-dialog-modal [aria-rowindex="${rowindex}"]'
+    build_btn: '#rule-based #btn-build'
+    tooltip: '.b-tooltip'
```

Comment:
Selenium noob here, I didn't see where this is used here ? If it's not used we should remove it.

Reply:
yes, you're right. I forgot to remove this `tooltip`